docs/en-US/component/statistic.md

```diff
@@ -0,0 +1,79 @@
+---
+title: statistic
+lang: en-US
+---
+
+# statistic
+
+Display statistics.
+
+## Basic usage
+
+:::demo To highlight a number or a group of numbers, such as statistical value, amount, and ranking, you can add elements such as icon and unit before and after the number and title.
+
+statistic/num
+
+:::
+
+## Countdown
+
+:::demo Set `timeIndices`Start the countdown.   Countdown component, support English and Chinese countdown, support to add other components control countdown.
+
+statistic/countdown
+
+:::
+
+## Card usage
+
+
+:::demo Card usage display, can be freely combined
+
+statistic/card
+
+:::
+
```

Comment:
## Suggest

It seems necessary to use prettier to clean up unnecessary whitespace or code.

Reply:
Thank you. I'll adjust it